MapboxSearch/sdk/src/androidTest/java/com/mapbox/search/SearchEngineIntegrationTest.kt

```diff
@@ -888,6 +888,49 @@ internal class SearchEngineIntegrationTest : BaseTest() {
         countDownLatch.await()
     }
 
+    @Test
+    fun testErrorBackendResponseSimpleFormat() {
```

Comment:
[nit] It would be cool to have such tests for `CategorySearchEngine` and `ReverseSearchEngine` too! 🙏